tech/languages/csharp/csharp-ide.md

```diff
@@ -6,27 +6,31 @@ order: 2
 
 # C# IDEs in Fedora
 
+## JetBrains Rider
+
+_.NET Core and Mono_
+
+JetBrains Rider is full featured IDE for all things C#, and even F# on .NET Core. If you are familiar with other JetBrains products, such as the IntelliJ or Resharper, you might like this one as well.
+
+[jetbrains.com/rider](http://jetbrains.com/rider)
```

Comment:
Please also note, that the commit message does reflect the added content- `JetBrains Rider` block.

Maybe you could just change the text to `update csharp IDEs page` or similar (alternatively you can specify the changes and/or just split them to muliple commits).

Reply:
Scratch this ^ my bad :) it's just rearranged, not added.

But that can be (optionally) noted in the commit as well :D

Reply:
Yeah sorry forgot to mention... I moved the bugged IDE's below what's de facto the only working C# IDE right now.